modules/services/ui/src/main/java/au/com/shiftyjelly/pocketcasts/ui/extensions/Fragment.kt

```diff
@@ -12,3 +13,12 @@ fun Fragment.openUrl(url: String) {
         Timber.e(e)
     }
 }
+
+@Suppress("DEPRECATION")
+fun Fragment.setupKeyboardModeResize() {
+    activity?.window?.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
+}
+
+fun Fragment.setupKeyboardModePan() {
+    activity?.window?.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN)
+}
```

Comment:
`SOFT_INPUT_ADJUST_RESIZE` is deprecated , since we're targeting 7.69 beta to be out today, I'm reusing the existing solution used on other screens to resize a fragment based on keyboard size.